db/controllers.js

```diff
@@ -9,4 +10,25 @@ function getAllTopics(req, res) {
     })
 }
 
-module.exports = {getAllTopics};
\ No newline at end of file
+function updateEndpointsDynamically(data) {
+    return fs.writeFile("/home/hashim/northcoders/backend/be-nc-news/endpoints.json", JSON.stringify(data, null, 2), "utf-8");
```

Comment:
This file path is not good. If someone else was to run your code it wouldn't work as this is a file path on your local machine!

All these need to be fixed